### DISCUSSION TITLE
my code isnt recognized for some reason, please help

### DISCUSSION THREAD

**Discussion by dr-ez** [openai/whisper#2327] [Q&A] [answered]
whisper: error: unrecognized arguments: --max_word_per_line 5
am trying to make a transcription which was done correctly but was timestamped per sentence which could get annoyingly long.
am trying to limit it to 5 words per line.
here is the code i tried to run
!whisper "to subtitles.mp3" --model large --language ar --word_timestamps True --max_word_per_line 5
![image](https://github.com/user-attachments/assets/68ba5b40-8628-48c5-a174-5f2386779500)

**Comment by misutoneko** [comment, marked as answer]
That's easy, it's just a typo. You're missing an "s" (word vs. words).

**Reply by dr-ez** [reply]
damn, sorry i guess being awake for 2 days makes you blind. XD
Thanks.
since I have you do you know if there is anyway to make it add highlight to the word by word being spoken?

**Reply by misutoneko** [reply]
No, sorry, haven't used that. I have an older version of Whisper which seems to have a command line switch --highlight_words, but I don't know how that's supposed to work (or why it was removed).
